docs/components/clerk-provider.mdx

```diff
@@ -93,9 +89,11 @@ The `<ClerkProvider>` component must be added to your React entrypoint.
-| `frontendApi`             | `string`                                                                              | The frontend API host for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page under the **Advanced** section.                                                                                                                                        |
-| `navigate?`               | `(to: string) => Promise<unknown> \| void`                                            | A function which takes the destination path as an argument and performs a "push" navigation.                                                                                                                               |
+| `publishableKey`          | `string`                                                                              | The Clerk publishable key for your instance. This can be found in your Clerk Dashboard on the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys)** page.                                                                                                                                                   |
+| `frontendApi` (deprecated)             | `string`                                                                              | Deprecated in favor of `publishableKey`.                                                                                                                                        |
+| `routerPush?` | `(to: string) => Promise<unknown> \| void`                                                                              | A function which takes the destination path as an argument and performs a "push" navigation.                                                                                                                                        |
+| `routerReplace?` | `(to: string) => Promise<unknown> \| void`                                                                              | A function which takes the destination path as an argument and performs a "replace" navigation.                                                                                                                                        |
+| `navigate?` (deprecated)               | `(to: string) => Promise<unknown> \| void`                                            | Deprecated in favor of `routerPush` or `routerReplace`, depending on your needs. Learn more in our [v4 to v5 upgrade guide](/docs/upgrade-guides/upgrading-from-v4-to-v5#replace-navigate-with-router-push-router-replace).                                                                                                                               |

```

Comment:
`navigate` isn't deprecated in v5, it got removed. So this row would need to live on the v4 docs